### PR TITLE
added CSS style to animal trading card

### PR DESCRIPTION
**Pull request by seb-lex**
Recreated animal trading card according to below prototype.

**Prototype:**
design-prototype

**End result:**
image

### CODE REVIEW

File: animal-trading-cards/normalize.css [hunk 1]
```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```
Comment (Martian2Lee) [inline animal-trading-cards/normalize.css:3]: mention it's from third party

File: animal-trading-cards/card.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```
Comment (Martian2Lee) [inline animal-trading-cards/card.html:1]: what changed

File: animal-trading-cards/style.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+     height: 0;
+}
+
+.clear {
+	clear: both;
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
```
Comment (Martian2Lee) [inline animal-trading-cards/style.css:22]: what's the change

**Review by Martian2Lee — CHANGES_REQUESTED** [review]

**Review by Martian2Lee — CHANGES_REQUESTED** [review]
please update title and add description

File: animal-trading-cards/card.html (hunk 2 above)
Comment (seb-lex) [reply animal-trading-cards/card.html:1]: https://github.com/sebz996/front-end-web-developer-course/pull/1/files#diff-d8b8a0922f69d22c15d389066732fda56250d5a3355531995a21ff37ab314ddbR12

File: animal-trading-cards/style.css (hunk 3 above)
Comment (seb-lex) [reply animal-trading-cards/style.css:22]: Added the following lines to pre-made template

https://github.com/sebz996/front-end-web-developer-course/pull/1/files#diff-48ddfca73e95f50ab8388e1f15a5f5b0df81d0137a98edcba12e00dbba71d961R24-R58

File: animal-trading-cards/normalize.css (hunk 1 above)
Comment (seb-lex) [reply animal-trading-cards/normalize.css:3]: done

https://github.com/sebz996/front-end-web-developer-course/pull/1/commits/2b97e312bd4557eb3223ac1692571df2544caf80

**Review by Martian2Lee — APPROVED** [review]
🚀

**Pull request merged**